app/helpers/admin/events_helper.rb

```diff
@@ -15,6 +15,14 @@ def admin_visit_requests_link(event, options = { class: 'item' })
         admin_event_visit_requests_path(event), options
     end
 
+    def admin_visitors_report_link(event)
+      return unless event.confirmation?
```

Comment:
I am not sure about this condition. What if event already started(`live`) and we need to print names one more time?
However it is a minor issue, that can be checked with Oksana or Anna after the merge.